specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+        "tenantId": {
+          "type": "string",
+          "description": "The AAD Tenant ID associated with the management group. For example, 00000000-0000-0000-0000-000000000000"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the management group."
+        }
+      }
+    },
+    "ListSubscriptionUnderManagementGroup": {
+      "description": "The details of all subscriptions under management group.",
+      "properties": {
+        "value": {
+          "description": "The list of subscriptions.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubscriptionUnderManagementGroup"
+          }
+        },
+        "nextLink": {
+          "description": "The URL to use for getting the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SubscriptionUnderManagementGroup": {
+      "description": "The details of subscription under management group.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the subscription.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000/subscriptions/0000000-0000-0000-0000-000000000001",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource.  For example, Microsoft.Management/managementGroups/subscriptions",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The stringified id of the subscription. For example, 00000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "properties": {
+          "title": "Properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SubscriptionUnderManagementGroupProperties"
+        }
+      }
+    },
+    "SubscriptionUnderManagementGroupProperties": {
+      "description": "The generic properties of subscription under a management group.",
+      "type": "object",
+      "properties": {
+        "tenant": {
+          "type": "string",
+          "description": "The AAD Tenant ID associated with the subscription. For example, 00000000-0000-0000-0000-000000000000"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the subscription."
+        },
+        "parent": {
+          "title": "Parent",
+          "$ref": "#/definitions/DescendantParentGroupInfo",
+          "x-nullable": true
+        },
+        "state": {
+          "type": "string",
+          "description": "The state of the subscription."
+        }
+      }
+    },
+    "ManagementGroup": {
+      "description": "The management group details.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
```

Comment:
>id [](http://example.com/codeflow?start=9&length=2)

Top-level properties (id, name, type, systemData, etc.) should come from v5 common types (either TrackedResource, ProxyResource, or Resource definitions)